ui/src/utils/axios.js

```diff
@@ -134,11 +134,6 @@ export default (callback, store, router) => {
                     variant: "error"
                 })
 
-                if(errorResponse.response.status === 401 &&
-                    store.getters["auth/isLogged"]){
-                    store.commit("auth/setExpired", true);
-                }
```

Comment:
This part was probably introduced by a wrong rebase, the setExpired doesn't even exist